build.psm1

```diff
@@ -848,6 +848,10 @@ function Start-PSPester {
         [switch]$IncludeFailingTest
     )
 
+    if (-not (Test-Path $Pester)) {
+        Write-Warning "Pester module not found. Make sure that the proper git submodules are installed by running:`r`n`r`ngit submodule update --init`r`n "
```

Comment:
Should this be an error instead? We would not be able to run tests without Pester.

Reply:
@adityapatwardhan I could change this to use `Write-Error, but now I'm wondering if we should use `throw` instead -- after all, there is no point in continuing at all if Pester can't be found.

Reply:
Actually if we follow the pattern from: https://github.com/DdWr/PowerShell/blob/3fdd9f1b639df805f895c56cf574f2039ec15a76/build.psm1#L415

We should do Write-Warning and then return.

Reply:
@adityapatwardhan Changes made.